### PR TITLE
Adding win devicelab bot results to dashboard. (#80974)

### PR DESCRIPTION
**Pull request by yusufm**
*Replace this paragraph with a description of what this PR is changing or adding, and why. Consider including before/after screenshots.*

*List which issues are fixed by this PR. You must list at least one issue.*

*If you had to change anything in the [flutter/tests] repo, include a link to the migration guide as per the [breaking change policy].*

## Pre-launch Checklist

- [X ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [X ] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [X ] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [X ] I signed the [CLA].
- [X ] I listed at least one issue that this PR fixes in the description above.
- [X ] I updated/added relevant documentation (doc comments with `///`).
- [X ] I added new tests to check the change I am making or feature I am adding, or Hixie said the PR is test-exempt.
- [X ] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

File: dev/prod_builders.json [hunk 1]
```diff
@@ -845,6 +845,48 @@
          "repo": "flutter",
          "task_name": "win_web_tool_tests",
          "flaky": true
+      },
```
Comment (CaseyHillers) [inline dev/prod_builders.json:848]: Can you remove `dev/devicelab/manifest.yaml` as well?

Comment (yusufm) [reply dev/prod_builders.json:848]: done.

**Review by christopherfujino — APPROVED** [review]
LGTM

File: dev/devicelab/manifest.yaml [hunk 2]
```diff
@@ -1,349 +0,0 @@
...
-
-  windows_chrome_dev_mode:
-    description: >
-      Run flutter web on the devicelab and hot restart.
-    stage: devicelab_win
-    required_agent_capabilities: ["windows/android"]
-
-  # iOS on-device tests
-
-  flutter_gallery_sksl_warmup__transition_perf_e2e_ios32:
```
Comment (CaseyHillers) [inline dev/devicelab/manifest.yaml:89]: There's various iOS tests missing, can you add those as well?

File: dev/prod_builders.json [hunk 3]
```diff
@@ -845,6 +845,48 @@
...
+        "flaky": false
+      },
+      {
+        "name": "Windows_android flutter_gallery_win__compile",
+        "repo": "flutter",
+        "task_name": "flutter_gallery_win__compile",
+        "flaky": false
+      },
+      {
+        "name": "Windows_android windows_chrome_dev_mode",
```
Comment (CaseyHillers) [inline dev/prod_builders.json:868]: Did we enable these builds in flutter/infra for this branch?

File: dev/devicelab/manifest.yaml (hunk 2 above)
Comment (yusufm) [reply dev/devicelab/manifest.yaml:89]: I assume those are not to be run on this branch?

File: dev/prod_builders.json (hunk 3 above)
Comment (yusufm) [reply dev/prod_builders.json:868]: I do not know how to check that.

Comment (christopherfujino) [reply dev/prod_builders.json:868]: Yep, the release branches copy the master builders, so this PR added these builders for all release branches: https://github.com/flutter/infra/pull/410/files

File: dev/devicelab/manifest.yaml (hunk 2 above)
Comment (CaseyHillers) [reply dev/devicelab/manifest.yaml:89]: They should be. These were missed from the devicelab migration

**Review by CaseyHillers — APPROVED** [review]
LGTM - Since release branches rely on MILO, it's okay not to add everything to prod_builders.json

**Comment by yusufm** [comment]
deleting the manifest_test.dart as well.

**Comment by yusufm** [comment]
Are the failures expected? Is this ok to merge?

**Comment by christopherfujino** [comment]
> Are the failures expected? Is this ok to merge?

Yeah, this is good to merge.

**Pull request merged**